package.json

```diff
@@ -36,16 +38,15 @@
     "@types/node": "^6.0.49",
     "@types/sinon": "^1.16.32",
     "codecov.io": "0.1.6",
-    "@dojo/cli": "^2.0.0-alpha.9",
-    "@dojo/loader": "^2.0.0-beta.9",
     "glob": "^7.0.3",
     "grunt": "~1.0.1",
-    "grunt-dojo2": ">=2.0.0-beta.36",
+    "grunt-dojo2": ">=2.0.0-beta.39",
     "intern": "^3.4.1",
     "istanbul": "^0.4.3",
     "mockery": "^1.7.0",
     "remap-istanbul": "^0.6.4",
     "sinon": "^1.17.5",
+    "tslint": "^4.0.0",
```

Comment:
OK. Removing `node_modules` did the trick. @agubler how did you get to be so smart?